package.json

```diff
@@ -153,6 +153,7 @@
     "should": "^13.2.3",
     "sinon": "^1.17.2",
     "source-map": "^0.4.4",
+    "supports-color": "^6.1.0",
     "temp-write": "^3.4.0",
```

Comment:
This seems unrelated to your changes?

Reply:
removed.

---
In reply to: [287893633](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r287893633) [](ancestors = 287893633)